sqlalchemy_imageattach/entity.py

```diff
@@ -568,14 +578,18 @@ def from_blob(self, blob, store=current_store):
                       :data:`~sqlalchemy_imageattach.context.current_store`
                       by default
         :type store: :class:`~sqlalchemy_imageattach.store.Store`
+        :param extra_args: additional arguments to pass to the model's constructor.
+        :type extra_args: :class:`list`
+        :param extra_kwargs: additional keyword arguments to pass to the model's constructor.
+        :type extra_kwargs: :class:`dict`
         :returns: the created image instance
         :rtype: :class:`Image`
 
         """
         data = io.BytesIO(blob)
-        return self.from_raw_file(data, store, original=True)
+        return self.from_raw_file(data, store, original=True, extra_args=extra_args, extra_kwargs=extra_kwargs)
```

Comment:
Please [limit lines to a maximum 79 characters][1] e.g.:

```python
return self.from_raw_file(data, store,
                          original=True,
                          extra_args=extra_args,
                          extra_kwargs=extra_kwargs)
```

[1]: https://www.python.org/dev/peps/pep-0008/#maximum-line-length